media/css/firefox/os/firefox-os-new.less

```diff
@@ -0,0 +1,960 @@
+
+    ul {
+        margin: 0;
+        padding: 0;
+        list-style: none;
+    }
+}
+/* class is added to fxos-hero-space when switching to demo mode */
+.trim-height {
+    height: 900px;
+}
+.phone {
+    position: relative;
+    top: 0;
+    margin: 0 auto;
+    opacity: 1;
+    .transition(all .7s);
+    .signup {
+        margin-bottom: 10px;
+    }
+    .signup-txt {
+        color: #BEE1F5;
+        margin: 0 auto;
+        width: 18em;
+        .font-size(1.4rem);
+        letter-spacing: -0.03em;
+
+        @media only screen and (max-width: 1024px) {
+            .font-size(1.2rem);
+        }
+    }
+}
+.scale {
+    position: absolute;
+    top: -824px;
+    left: 353px;
+    .transform(scale(.3, .3));
+    opacity: 0;
+}
+.fxos-cta,
+.phone-container {
+    position: relative;
+    display: inline-block;
+    width: 45%;
+    vertical-align: middle;
+
+    @media only screen and (max-width: @breakTablet) {
+        display: block;
+        width: 100%;
+    }
+}
+.phone-container {
+    img {
+        width: 520px;
+        height: 394px;
+        .transition(all 1s);
+
+        @media only screen and (max-width: 1600px) {
+            width: 520px;
+            height: 394px;
+        }
+        @media only screen and (max-width: 1366px) {
+            width: 500px;
+            height: 379px;
+        }
+        @media only screen and (max-width: 1024px) {
+            width: 400px;
+            height: 303px;
+        }
+        @media only screen and (max-width: 768px) {
+            width: 400px;
+            height: 303px;
+        }
+        @media only screen and (min-width: @breakMobileLandscape) and (max-width: @breakTablet) {
+            width: 460px;
+            height: 349px;
+        }
+        @media only screen and (max-width: @breakMobile) {
+            width: 300px;
+            height: 228px;
+        }
+    }
+}
+.try-homescreen,
+.try-swipe {
+    display: none;
+    span {
+        .visually-hidden();
+    }
+}
+
+.try-homescreen {
+    position: absolute;
+    top: 320px;
+    left: 210px;
+    background: transparent url(/media/img/firefox/os-new/icons/arrows-vertical.png) 0 0 no-repeat;
+    width: 134px;
+    height: 134px;
+}
+.try-swipe {
+    position: absolute;
+    top: 390px;
+    left: 950px;
+    background: transparent url(/media/img/firefox/os-new/icons/arrows-horizontal.png) 0 0 no-repeat;
+    width: 134px;
+    height: 134px;
+}
+.throbber {
+    position: absolute;
+    top: 75px;
+    left: 77px;
+    display: inline-block;
+    background-color: white;
+    border-radius: 100px;
+    width: 100px;
+    height: 100px;
+    opacity: 0;
+    .transform(scale(0, 0));
+
+    -webkit-animation-name: throb;
+    -webkit-animation-duration: 4s;
+    -webkit-animation-iteration-count: infinite;
+    -webkit-animation-timing-function: ease-in-out;
+
+    animation-name: throb;
+    animation-duration: 4s;
+    animation-iteration-count: infinite;
+    animation-timing-function: ease-in-out;
```

Comment:
Could use `.animation` mixin to avoid vendor prefixes, but again, not critical for launch.